sources/web/datalab/polymer/modules/bigquery-file-manager/bigquery-file-manager.ts

```diff
@@ -12,6 +12,13 @@
  * the License.
  */
 
+type ListDatasetsResponse = gapi.client.bigquery.ListDatasetsResponse;
```

Comment:
Much cleaner.
FYI, I'm waiting on [this PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/19083) so we can use the gapi types from a declaration file that's automatically generated from the sources.

Reply:
Yes, that will be nice.